src/System.Buffers.Experimental/System/Range.cs

```diff
@@ -9,7 +9,7 @@
 namespace System.Buffers.Experimental
 {
     // TODO: consider allowing Last > First. Ennumeration will count down.
-    public readonly struct Range : IEnumerable<int>
+    public readonly struct RangeLab : IEnumerable<int>
```

Comment:
As a quick (temporary) fix, I renamed the Range type within corefxlab to RangeLab. We need to remove it and update to use the new System.Range type: https://github.com/dotnet/corefxlab/issues/2628

cc @tarekgh